lib/languages.ts

```diff
@@ -43,6 +43,30 @@ type DefKeys =
 type LanguageDefinition = Pick<Language, DefKeys>;
 
 const definitions: Record<LanguageKey, LanguageDefinition> = {
+    coccinelle_for_c: {
+        name: 'C with Coccinelle 🐞',
```

Comment:
Could we drop the emojis perhaps? I don't think any of the other languages use emojis

Reply:
I introduced the ladybug emoji there to have it occur in the languages menu.
The ladybug ("Coccinelle" in French) is the mascotte animal of Coccinelle.
Given that Coccinelle works on one language among C and C++ plus its own language, having the ladybug *and* the language icon was a good mnemonic.
If that's problematic, I can remove this emoji, and perhaps produce a language icon containing a ladybug somehow, in addition to the target language.

Reply:
I removed the emojis; please see https://github.com/compiler-explorer/compiler-explorer/pull/7001/commits/72ec30e26e62db6e2da4ba8b71300e7dc261605e

Reply:
I see, I don't think it's that big of a deal but I figured it would be best to stay consistent with the rest of the language list. Thanks